### ISSUE TITLE
@nivo/line: When using slices, the x/y positioning for the slice & tooltip does not update

### ISSUE THREAD

**Issue by garand** [plouc/nivo#1304]
I am using a ResponsiveLine with `x` slices enabled, and also a custom `sliceTooltip`. When my project is in `watch` mode locally, hovering on the chart and moving left/right, up/down moves the slice and tooltip with the mouse. However when I build the project for production and run it, the position of the tooltip wherever I enter the chart does not update when moving the mouse over the chart.

At first this seems like a built process issue (and it might be?) but when using 0.64.0 the issue does not exist. It was only after updating to 0.65.0 and 0.65.1 that it started happening.

Unfortunately I can't seem to reproduce this in a CodeSandbox.

![nivo:line](https://user-images.githubusercontent.com/82437/100021089-ebe2c980-2dae-11eb-957f-bba456cdf3be.gif)

**Comment by plouc** [comment]
@garand, not easy to spot, but I think that's a duplicate of https://github.com/plouc/nivo/issues/1290, you can apply the fix suggested there until `react-spring` release a fix.

**Comment by FrancoPaez** [comment]
I have the same problem, but from locally, were you able to solve it? It does not work for me in any graph :(
